src/plugins/shared_ux/public/components/index.ts

```diff
@@ -25,6 +25,12 @@ export const LazySolutionToolbarButton = React.lazy(() =>
   }))
 );
 
+export const LazySolutionToolbarButtonProps = React.lazy(() =>
+  import('./toolbar/index').then(({ SolutionToolbarButtonProps }: any) => ({
```

Comment:
Two things: first, this should exist.

`LazySolutionToolbarButtonProps` is a Typescript interface, so there's no lazy load necessary-- types are removed at compile time.

Second, as @majagrubic pointed out: no need to tack `/index` onto this import.